c2rust-refactor/src/scripting.rs

```diff
@@ -135,16 +151,255 @@ impl<'lua> IntoLuaAst<'lua> for P<ast::Expr> {
                             )));
                         }
                     }
-                }
-                _ => {
-                    return Err(LuaError::external(format!(
-                        "{:?} is not yet implemented",
-                        expr.node
-                    )));
-                }
+                },
+                ExprKind::Box(boxed) => {
+                    ast.set("kind", "Box")?;
+                    ast.set("boxed", boxed.into_lua_ast(ctx, lua_ctx)?)?;
+                },
+                ExprKind::ObsoleteInPlace(_, _) => {
+                    ast.set("kind", "ObsoleteInPlace")?;
+
+                },
+                ExprKind::Array(values) => {
+                    let vals: LuaResult<Vec<_>> = values
+                        .into_iter()
+                        .map(|v| v.into_lua_ast(ctx, lua_ctx))
+                        .collect();
+
+                    ast.set("kind", "Array")?;
+                    ast.set("values", iter_to_lua_array(vals?.into_iter(), lua_ctx)?)?;
+                },
+                ExprKind::Call(path, args) => {
+                   let args: LuaResult<Vec<_>> = args
+                        .into_iter()
+                        .map(|v| v.into_lua_ast(ctx, lua_ctx))
+                        .collect();
+
+                    ast.set("kind", "Call")?;
+                    ast.set("path", path.into_lua_ast(ctx, lua_ctx)?)?;
+                    ast.set("args", iter_to_lua_array(args?.into_iter(), lua_ctx)?)?;
+                },
+                ExprKind::MethodCall(_, _) => {
+                    ast.set("kind", "MethodCall")?;
+
+                },
+                ExprKind::Tup(_) => {
+                    ast.set("kind", "Tup")?;
+
```

Comment:
I think at some point in the future we'll want to enable access to the tuple fields. Maybe add a `FIXME` or `TODO` comment here for now (unless you're still implementing this)?